js/base-util.js

```diff
@@ -174,7 +174,9 @@ function baseUtil () {
 	d20plus.ut.addAllCss = () => {
 		d20plus.ut.log("Adding CSS");
 
-		const targetSheet = window.document.styleSheets[0];
+		$(window.document.head).append("<style id=\"b20-all-css\"></style>");
```

Comment:
seems a bit clunky to add the thing and then find it; how about e.g.

```js
const ele = document.createElement("style");
document.head.appendChild(ele);
const targetSheet = ele.sheet
```

Reply:
ooh I didn't know we could do that